core/build.gradle

```diff
@@ -7,7 +7,7 @@ dependencies {
     implementation 'com.github.bumptech.glide:glide:4.12.0'
 
     // https://mvnrepository.com/artifact/org.bitcoinj/bitcoinj-core
-    api 'org.bitcoinj:bitcoinj-core:0.15.10'
+    api 'org.bitcoinj:bitcoinj-core:0.16.2'
```

Comment:
I Updated bitcoin library, cause it's used old Protobuf version, and we had conflicts via merge them.